Source/Shared/BPXCTestFile.m

```diff
@@ -129,15 +130,19 @@ + (instancetype)BPXCTestFileFromDictionary:(NSDictionary *)dict
     if ([testBundlePath rangeOfString:TESTHOST].location != NSNotFound) {
         testBundlePath = [testBundlePath stringByReplacingOccurrencesOfString:TESTHOST withString:testHostPath];
     } else if ([testBundlePath rangeOfString:TESTROOT].location != NSNotFound) {
-        testHostPath = [testHostPath stringByDeletingLastPathComponent]; //remove /bluepill
-        NSString *temp = [testHostPath stringByDeletingLastPathComponent]; //remove /iphonesimulator
-        //extract app name with .app extension
-        NSRange dotPosition = [testHostBundleIdentifier rangeOfString:@"." options:NSBackwardsSearch];
-        NSString *appName = [testHostBundleIdentifier substringFromIndex:dotPosition.location + 1];
-        NSString *appNameWithExtension = [appName stringByAppendingString:@".app"];
-        //append the app name to the direction
-        testHostPath = [testHostPath stringByAppendingString:appNameWithExtension];
-        testBundlePath = [testBundlePath stringByReplacingOccurrencesOfString:TESTROOT withString:temp];
+        if ([testHostPath rangeOfString:PLATFORMS].location != NSNotFound) {
+            
```

Comment:
any handling here?

Reply:
This is actually from a different PR... I'm not sure how I ended up committing it here. I'll finish it and piggyback my fix on this PR.